src/lib/shared/signature.ts

```diff
@@ -0,0 +1,42 @@
+import { babyjubjub } from "babyjubjub-ecdsa";
+
+export const generateSignatureKeyPair = (): {
```

Comment:
Can we make a ticket to eventually move this to [babyjubjub-ecdsa](https://github.com/jubmoji/babyjubjub-ecdsa/tree/main)?